### DISCUSSION TITLE
How to resolve this rendering issue?

### DISCUSSION THREAD

**Discussion by muditbhargava66** [alshedivat/al-folio#2431] [Q&A] [answered]
<img width="1800" alt="Screenshot 2024-05-21 at 6 15 47 PM" src="https://github.com/alshedivat/al-folio/assets/34854747/bcac0ecb-1f93-4c87-b712-6904b9884fd9">

Link : https://muditbhargava66.github.io/

**Comment by george-gca** [comment, marked as answer]
Check our [FAQ](https://github.com/alshedivat/al-folio/blob/master/FAQ.md#my-webpage-works-locally-but-after-deploying-it-is-not-displayed-correctly-css-and-js-are-not-loaded-properly-how-do-i-fix-that).

**Reply by muditbhargava66** [reply]
I've tried all the FAQs and recommended steps, but I'm still encountering the rendering error.

**Reply by muditbhargava66** [reply]
Also, I have another site hosted on the gh-pages. Is it encountering issues with that? Also, both sites have different domains.

**Reply by george-gca** [reply]
Since I can't see your code I can only assume things. Have you left `baseurl` blank? This is the most common error that causes this issue.

**Reply by george-gca** [reply]
This should not be a problem as long as the 2 pages have different paths, which in the case of our template is set in the `baseurl`.

**Reply by muditbhargava66** [reply]
I have made the repository public: https://github.com/muditbhargava66/muditbhargava66.github.io. It would be very helpful if you could check and let me know what went wrong.

<img width="1800" alt="Screenshot 2024-05-21 at 6 40 41 PM" src="https://github.com/alshedivat/al-folio/assets/34854747/31013dca-70ad-40ea-9d5c-fb65290df874">

**Reply by george-gca** [reply]
The problem is exactly what I linked you for in our [FAQ](https://github.com/alshedivat/al-folio/blob/master/FAQ.md#my-webpage-works-locally-but-after-deploying-it-is-not-displayed-correctly-css-and-js-are-not-loaded-properly-how-do-i-fix-that). You have to leave `baseurl` empty.

**Reply by muditbhargava66** [reply]
It worked! I guess I missed that basic step. Also, can the maintainers add a photo-based `INSTALL.md`?

Thank you for helping @george-gca!

**Reply by george-gca** [reply]
What do you mean by photo based? Also I am not sure that adding another level of explanation would help. See, this specific step is both explained in our:
- [INSTALL.md - Recommended Approach step 3](https://github.com/alshedivat/al-folio/blob/master/INSTALL.md#recommended-approach)
- the problem caused if this is not followed in our [FAQ.md - My webpage works locally. But after deploying, it is not displayed correctly (CSS and JS are not loaded properly). How do I fix that?](https://github.com/alshedivat/al-folio/blob/master/FAQ.md#my-webpage-works-locally-but-after-deploying-it-is-not-displayed-correctly-css-and-js-are-not-loaded-properly-how-do-i-fix-that)
- also lots of people have already opened issues (#2382) or questions (#2313, #2074) about this, and they all have been answered.

Usually you can just search for your error message or the description of what is your error in GitHub limiting to our repo and the answer has already been provided.
